scripts/build_container.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e -o pipefail
+
+MUNGED_BRANCH=$(echo $CIRCLE_BRANCH | tr '/' '_')
+docker build . \
+  -t quay.io/aipg/${CIRCLE_PROJECT_REPONAME}:${CIRCLE_SHA1} \
+  -t quay.io/aipg/${CIRCLE_PROJECT_REPONAME}:${MUNGED_BRANCH} \
+  --build-arg BUILD_DATE=$(date -u +"%Y-%m-%dT%H:%M:%SZ") \
```

Comment:
We need an optional detection for the PIPBOT_PEM build arg for containers with secrets:
```
 --build-arg PIPBOT_PEM="${PIPBOT_PEM}"
```

Maybe it can always be passed in? anyhow reminder so we don't forget it.